downstream/modules/platform/ref-controller-amazon-web-services.adoc

```diff
@@ -0,0 +1,109 @@
+[id="controller-amazon-web-services"]
+
+= Amazon Web Services EC2
+
+----
```

Comment:
Can you add a code block header before each [literal, options="nowrap" subs="+attributes"]

It allows you to include attributes and avoid the copy/paste copying any $ or # signs.

Reply:
Done